Lib/ufo2ft/filters/dottedCircleFilter.py

```diff
@@ -130,7 +130,7 @@ def check_dotted_circle(self):
         dotted_circle = next((g.name for g in font if 0x25CC in g.unicodes), None)
```

Comment:
This should then also iter over the glyphSet I suppose?

Reply:
i think so, otherwise you may get a KeyError